lib/Doctrine/ruleset.xml

```diff
@@ -275,52 +265,50 @@
     <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration">
         <properties>
             <property name="enableEachParameterAndReturnInspection" value="true"/>
-            <property name="traversableTypeHints" type="array" value="Doctrine\Common\Collections\Collection"/>
-            <property
-                name="usefulAnnotations"
-                type="array"
-                value="
-                    @after,
-                    @afterClass,
-                    @AfterMethods,
-                    @Attribute,
-                    @Attributes,
-                    @before,
-                    @beforeClass,
-                    @BeforeMethods,
-                    @covers,
-                    @coversDefaultClass,
-                    @coversNothing,
-                    @dataProvider,
-                    @depends,
-                    @deprecated,
-                    @doesNotPerformAssertions,
-                    @Enum,
-                    @expectedDeprecation,
-                    @expectedException,
-                    @expectedExceptionCode,
-                    @expectedExceptionMessage,
-                    @expectedExceptionMessageRegExp,
-                    @group,
-                    @Groups,
-                    @IgnoreAnnotation,
-                    @internal,
-                    @Iterations,
-                    @link,
-                    @ODM\,
-                    @ORM\,
-                    @requires,
-                    @Required,
-                    @Revs,
-                    @runInSeparateProcess,
-                    @runTestsInSeparateProcesses,
-                    @see,
-                    @Target,
-                    @test,
-                    @throws,
-                    @uses
-                "
-            />
+            <property name="traversableTypeHints" type="array">
+                <element value="Doctrine\Common\Collections\Collection"/>
+            </property>
+            <property name="usefulAnnotations" type="array">
```

Comment:
@carusogabriel Btw you can use `allAnnotationsAreUseful` now, see https://github.com/slevomat/coding-standard#slevomatcodingstandardtypehintstypehintdeclaration-

Reply:
That's probably for a separate discussion. But given that we already use `forbiddenAnnotations`, we may consider this change and just expand the forbidden list for any of those that get removed by this rule.